<launch>
    <node name="data_logger_node" pkg="data_logger" type="data_logger" output="screen">
        <rosparam command="load" file="$(find data_logger)/config/data_logger.yaml" /> <!--Load parameters from config files-->
        <param name="log_filename" value="$(find waypoint_loader)/data/track_data/virtual_vehicle_2.csv" />
    </node>
</launch>
